offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/DependencyManager.groovy

```diff
@@ -150,46 +170,79 @@ class DependencyManager {
 
     private void setupDbConnections() {
         try {
-
             String user = Objects.requireNonNull(configurationManager.getMysqlUser(), "Mysql user missing.")
             String password = Objects.requireNonNull(configurationManager.getMysqlPass(), "Mysql password missing.")
             String host = Objects.requireNonNull(configurationManager.getMysqlHost(), "Mysql host missing.")
             String port = Objects.requireNonNull(configurationManager.getMysqlPort(), "Mysql port missing.")
             String sqlDatabase = Objects.requireNonNull(configurationManager.getMysqlDB(), "Mysql database name missing.")
 
             DatabaseSession.init(user, password, host, port, sqlDatabase)
-            customerDbConnector = new PersonDbConnector(DatabaseSession.getInstance())
-            productsDbConnector = new ProductsDbConnector(DatabaseSession.getInstance())
-            offerDbConnector = new OfferDbConnector(DatabaseSession.getInstance(),
-                    customerDbConnector, productsDbConnector)
-            projectDbConnector = new ProjectDbConnector(DatabaseSession.getInstance(), customerDbConnector)
-
+            PersonDbConnector personDbConnector = new PersonDbConnector(DatabaseSession.getInstance())
+            createPersonDataSource = personDbConnector
+            searchPersonDataSource = personDbConnector
+            createAffiliationDataSource = personDbConnector
+            listAffiliationsDataSource = personDbConnector
+            listPersonsDataSource = personDbConnector
+
+            ProductsDbConnector productsDbConnector = new ProductsDbConnector(DatabaseSession.getInstance())
+            archiveProductDataSource = productsDbConnector
+            createProductDataSource = productsDbConnector
+            copyProductDataSource = productsDbConnector
+            listProductsDataSource = productsDbConnector
+
+            /* Currently life.qbic.portal.offermanager.dataresources.offers.OfferDbConnector
+             *  cannot be decoupled by interfaces from
+             *  life.qbic.portal.offermanager.dataresources.persons.PersonDbConnector nor
+             *  life.qbic.portal.offermanager.dataresources.products.ProductsDbConnector
+            */
```

Comment:
I think we dont need these comments

Reply:
I think they might offer help because you have to modify those classes and cannot just re-wire them.